plugins/win-dshow/win-dshow.cpp

```diff
@@ -980,11 +980,13 @@ bool DShowInput::UpdateVideoConfig(obs_data_t *settings)
 	     "\tresolution: %dx%d\n"
 	     "\tflip: %d\n"
 	     "\tfps: %0.2f (interval: %lld)\n"
-	     "\tformat: %s",
+	     "\tformat: %s\n"
+	     "\tbuffering: %s",
 	     obs_source_get_name(source), (const char *)name_utf8,
 	     (const char *)path_utf8, videoConfig.cx, videoConfig.cy_abs,
 	     (int)videoConfig.cy_flip, fps, videoConfig.frameInterval,
-	     formatName->array);
+	     formatName->array,
+	     obs_source_async_unbuffered(source) ? "disabled" : "enabled");
 
 	SetupBuffering(settings);
```

Comment:
It looks like this is logged before the buffering is actually applied to the source. Perhaps checking the settings would be better, i.e. `obs_data_get_int(settings, BUFFERING_VAL)`

Reply:
The dshow logs always print twice (prior to my pr as well). Before your proposed change it would print the prior value, and then the updated one. Now it prints the new value in the first and second print.

Not sure if its unintentional that its printing twice (before and after?), but yeah, I'll happily change it.

Reply:
hmm, it seems to me that `BUFFERING_VAL` is outputting what the user has selected (auto, enable or disable).
I find that less valuable than whether the buffering is in use or not.

If the user set auto, then you wont actually know based on the log whether its in use or not.
I believe the original implementation would solve `auto` into whether buffering is enabled or not.
It might happen on the 2nd print, but it would do it (I believe).

Reply:
Ok, so I think that this should take care of the concern while still doing what I intended it to do.

Spent some time looking at the double logging (too complicated for me), but seeing as it was always there, I feel like that will have to be for someone else to dig into, at some point, but is unrelated to the PR.